src/logic.js

```diff
@@ -1,28 +1,30 @@
+/* eslint-disable */
```

Comment:
- [ ] Please don't disable linters. They are helpful in ensuring that your code adheres to best practices and coding conventions👍

Reply:
@codepantha, precisely it cannot solve that is why I sent you the following message:

> In order to implement the functionality of the clear all button, I had to make a cyclic call in order to refresh the view and show the changes.

If you could help me with an idea of how to solve it I would greatly appreciate it.